Gemfile.lock

```diff
@@ -82,6 +82,7 @@ GEM
     docile (1.1.5)
     equalizer (0.0.11)
     erubis (2.7.0)
+    excon (0.45.4)
```

Comment:
How interesting, it used an ancient version I had install. There was no reason for this. Will update.